parser/model/ddl.go

```diff
@@ -426,18 +436,20 @@ func (sub *SubJob) FromProxyJob(proxyJob *Job, ver int64) {
 type JobMeta struct {
 	SchemaID int64 `json:"schema_id"`
 	TableID  int64 `json:"table_id"`
-	// Query string of the ddl job.
+	// Type is the DDL job's type.
+	Type ActionType `json:"job_type"`
+	// Query is the DDL job's SQL string.
 	Query string `json:"query"`
 	// Priority is only used to set the operation priority of adding indices.
 	Priority int `json:"priority"`
 }
 
 // BackfillMeta is meta info of the backfill job.
 type BackfillMeta struct {
-	PhysicalTableID int64  `json:"physical_table_id"`
-	IsUnique        bool   `json:"is_unique"`
-	EndInclude      bool   `json:"end_include"`
-	ErrMsg          string `json:"err_msg"`
+	PhysicalTableID int64         `json:"physical_table_id"`
+	IsUnique        bool          `json:"is_unique"`
+	EndInclude      bool          `json:"end_include"`
+	Error           *terror.Error `json:"err"`
```

Comment:
No usage?

Reply:
Using it in the function of `GetBackfillErr`.